sktime/classification/deep_learning/inceptiontime.py

```diff
@@ -41,6 +41,16 @@ class InceptionTimeClassifier(BaseDeepClassifier):
 
     Adapted from the implementation from Fawaz et. al
     https://github.com/hfawaz/InceptionTime/blob/master/classifiers/inception.py
+
+    Examples
+    --------
+    >>> from sktime.classification.deep_learning.inceptiontime import InceptionTimeClassifier # doctest: +SKIP # noqa
```

Comment:
What is this `noqa` for? Is it for line length? Please fix by splitting into multiple lines instead of adding noqa.

Also, I think doctest need not be skipped in this line, but it's okay to have as well.